### ISSUE TITLE
How can I install thorium avx2 with git in arch?

### ISSUE THREAD

**Issue by xisnamh** [Alex313031/thorium#739]
I know that the builds are compiled in .zip and .appimage to download and install.

But I would like to use git and select the version to install, thanks!

And another question, I have read that the "normal" version of Thorium already comes with the avx2 flags? could be? or did I understand it wrong?

**Comment by gz83** [comment]
The versions for different architectures are clearly distinguished in the file names. If you need AVX2, please choose the version with AVX2 in the file name.

**Comment by xisnamh** [comment]
Yes, I know, I wonder how you can download/install the avx2 version with git, for example? thanks !!

**Comment by gz83** [comment]
You can download it from the releases page in the Thorium repo.

**Comment by salehabdullah216** [comment]
just copy the file link in the releases page and use wget or use the aur
